libraries/Microsoft.Bot.Builder.AI.QnA/Models/AnswerSpan.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    public class AnswerSpan
```

Comment:
better call this class as 'AnswerSpanResponse'

align with Request / Response format.

Reply:
Needs more changes.. may be next time.